packages/autocertifier/src/RestInterface.ts

```diff
@@ -0,0 +1,15 @@
+import { CertifiedSubdomain } from '@streamr/autocertifier-client'
+import { Session } from '@streamr/autocertifier-client'
+
+export interface RestInterface {
+    createSession(): Promise<Session>
+    
+    createNewSubdomainAndCertificate(ipAddress: string, port: string, streamrWebSocketPort: string, 
```

Comment:
What are `streamrWebSocketPort` and `streamrWebSocketCaCert`

Reply:
`streamrWebSocketCaCert` is removed, added TODO comment about `streamrWebSocketPort`